src/index.js

```diff
@@ -1,7 +1,7 @@
 import './style.css';
-import scores from './scoresList.js';
 
 const tableBody = document.querySelector('tbody');
+const requestURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/XlbFBk8CtDvgI6FEd2iz/scores/';
 function scoreElement(name, score) {
   const tableRow = document.createElement('tr');
```

Comment:
- [ ] Kindly make use of arrow functions instead of the function keyword to follow ES6 best practices.